exercism/hamming/hamming.spec.js

```diff
@@ -1,40 +1,40 @@
 var Hamming = require('./hamming');
 
-describe('Hamming', function () {
+describe('Hamming', function() {
   var hamming = new Hamming();
 
-  it('no difference between identical strands', function () {
+  it('no difference between identical strands', function() {
     expect(hamming.compute('A', 'A')).toEqual(0);
   });
 
-  xit('complete hamming distance for single nucleotide strand', function () {
-    expect(hamming.compute('A','G')).toEqual(1);
+  it('complete hamming distance for single nucleotide strand', function() {
+    expect(hamming.compute('A', 'G')).toEqual(1);
   });
 
-  xit('complete hamming distance for small strand', function () {
-    expect(hamming.compute('AG','CT')).toEqual(2);
+  it('complete hamming distance for small strand', function() {
+    expect(hamming.compute('AG', 'CT')).toEqual(2);
   });
 
-  xit('small hamming distance', function () {
-    expect(hamming.compute('AT','CT')).toEqual(1);
+  it('small hamming distance', function() {
+    expect(hamming.compute('AT', 'CT')).toEqual(1);
   });
 
-  xit('small hamming distance in longer strand', function () {
+  it('small hamming distance in longer strand', function() {
     expect(hamming.compute('GGACG', 'GGTCG')).toEqual(1);
   });
 
-  xit('large hamming distance', function () {
+  it('large hamming distance', function() {
     expect(hamming.compute('GATACA', 'GCATAA')).toEqual(4);
   });
 
-  xit('hamming distance in very long strand', function () {
+  it('hamming distance in very long strand', function() {
     expect(hamming.compute('GGACGGATTCTG', 'AGGACGGATTCT')).toEqual(9);
   });
```

Comment:
Is there a reason you expect big strands to test/behave different than small strands? This seems unnecessary.